From: Nicolas Dandrimont <olasd@debian.org>
Date: Tue, 9 Oct 2018 15:38:25 +0200
Subject: update intersphinx links for local access

---
 docs/source/sphinx.toml | 7 +++----
 1 file changed, 3 insertions(+), 4 deletions(-)

diff --git a/docs/source/sphinx.toml b/docs/source/sphinx.toml
index d731a6f..d9adcda 100644
--- a/docs/source/sphinx.toml
+++ b/docs/source/sphinx.toml
@@ -18,7 +18,6 @@ extensions = [
     "sphinx.ext.graphviz",
     "sphinxcontrib.jquery",
     "IPython.sphinxext.ipython_console_highlighting",
-    "IPython.sphinxext.ipython_directive",
     "sphinx.ext.napoleon",  # to preprocess docstrings
     "github",  # for easy GitHub links
     "magics",
@@ -30,10 +29,10 @@ modindex_common_prefix = ["IPython."]
 [intersphinx_mapping]
 python = { url = 'https://docs.python.org/3/', fallback = '' }
 rpy2 = { url = 'https://rpy2.github.io/doc/latest/html/', fallback = '' }
-jupyterclient = { url = 'https://jupyter-client.readthedocs.io/en/latest/', fallback = '' }
-jupyter = { url = 'https://jupyter.readthedocs.io/en/latest/', fallback = '' }
+jupyterclient = { url = 'https://jupyter-client.readthedocs.io/en/latest/', fallback = '/usr/share/doc/python-jupyter-client-doc/html/objects.inv' }
+jupyter = { url = 'https://jupyter.readthedocs.io/en/latest/', fallback = '/usr/share/doc/python-jupyter-core-doc/html/objects.inv' }
 jedi = { url = 'https://jedi.readthedocs.io/en/latest/', fallback = '' }
-traitlets = { url = 'https://traitlets.readthedocs.io/en/latest/', fallback = '' }
+traitlets = { url = 'https://traitlets.readthedocs.io/en/latest/', fallback = '/usr/share/doc/python-traitlets-doc/html/objects.inv' }
 ipykernel = { url = 'https://ipykernel.readthedocs.io/en/latest/', fallback = '' }
 prompt_toolkit = { url = 'https://python-prompt-toolkit.readthedocs.io/en/stable/', fallback = '' }
 ipywidgets = { url = 'https://ipywidgets.readthedocs.io/en/stable/', fallback = '' }
